src/controllers/shop/checkout-item.controller.js

```diff
@@ -10,14 +10,23 @@ const checkoutItem = async (req, res) => {
       attributes: { exclude: ["password"] },
     });
 
-    const item = req.body;
+    const items = req.body;
 
     const transactions = [];
 
-    //for (const item of items) {
-      const { product_id, qty } = item;
+    for (const item of items) {
+      const { cart_id, qty } = item;
 
-      const product = await Product.findOne({ where: { id: product_id } });
+      const cart = await Cart.findOne({
+        where: { id: cart_id },
+      });
+      console.log(cart.id);
+      console.log(typeof cart.id);
+      if (!cart) {
+        return response(res, 404, false, "Cart not found", null);
+      }
+
+      const product = await Product.findOne({ where: { id: cart.product_id } });
```

Comment:
Di patch dengan cart yang sudah di-checkout, sehingga tidak terjadi duplikasi dan tidak menyebabkan data anomaly serta patch response
![image](https://github.com/Belega-Village-UNUD/backend-api-edb/assets/75294452/f84f552c-62f1-47bc-8576-ef95d4643f31)


![image](https://github.com/Belega-Village-UNUD/backend-api-edb/assets/75294452/df7f03d0-8947-4129-a6d9-e87d5427ac33)

Reply:
refer to commit: https://github.com/Belega-Village-UNUD/backend-api-edb/pull/51/commits/5dc0312c5a451ca44dfedd2a2349354ef8a3321e